src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -44,7 +44,164 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: GET
 
+  #
+  # Single Certificate
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/applications/{applicationId}':
```

Comment:
I'm not sure about this path ... it looks a lot like getApplicationById from VCA 😜 
Are we getting an application? or are we getting a certificate? If we are getting a certificate is it a full certificate or a certificate summary in some way? Perhaps that differentiator (cert vs. cert summary) doesnt matter so much in the path?, but I think the path should be different from getApplicationById .... 🤔 

What do you think?

Reply:
@NathanRussellValtech , I wasn't sure actually for the url and just followed the way we did it in the VCA where the path does not refer to the voter card application itself and we assume it would return a voter card application. In here I thought we might also infer this would return a certificate (or certificate summary), hence the implementation.

After your suggestion I changed it to `certificates/eros/{eroId}/applications/{applicationId}`. 

Does this make sense?

Reply:
After discussions, this is the final agreed endpoint: `/eros/{eroId}/certificates/applications/{applicationId}`